src/get-started/install/help.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Install help
+description: This page describes some common installation issues new Flutter users have run into and offers suggestions to resolve them.
+---
+
+This page describes some common installation issues new Flutter users have run
+into and offers suggestions to resolve them. Included at the bottom of the page
+is a list of [community support channels][] you can reach out to if you don’t
+find answers from this page. To add a topic to this page or make a correct, you
+may file an issue or a pull request using the buttons at the top of the doc.
+
+## Get the Flutter SDK
+
+
+### Unable to find the `flutter` command
+
+__What does this issue look like?__
+
+When you try to run the `flutter` command, the console fails to find it. On
+Windows, the error usually looks like this:
```

Comment:
@sfshaza2 The error message is Windows specific, should we keep "On Windows" from the original text?

Reply:
Well, the error exists on either platform: Windows or Unix. The error text does vary a bit, of course, but it's a pretty minor nit. We usually provide Unix errors and often don't bother to even include the Windows version. Any way, maybe I'm not the best person to address this, but it seems very minor. It would be more important if the error were ONLY on Windows or ONLY on Unix.

Reply:
I adopted your suggestion and added a line below the error message to point out it might look slightly different on macOS and Linux. PTAL.